lib/killbill/helpers/active_merchant/utils.rb

```diff
@@ -1,3 +1,6 @@
+require 'thread'
+require 'thread_safe'
```

Comment:
Yes, it's in development only because of the tests. These dependencies are required only for ActiveMerchant plugins, not for other Kill Bill plugins (e.g. notification or invoice plugin).

The dependencies for such payment plugins are generated from: https://github.com/killbill/killbill-plugin-framework-ruby/blob/master/generators/active_merchant/templates/plugin.gemspec.rb (we need to keep both in sync unfortunately).